finding/finding.go

```diff
@@ -131,16 +240,37 @@ func (f *Finding) WithRemediationMetadata(values map[string]string) *Finding {
 	if f.Remediation != nil {
 		// Replace all dynamic values.
 		for k, v := range values {
+			// Replace metadata.
 			f.Remediation.Text = strings.Replace(f.Remediation.Text,
-				fmt.Sprintf("${{ %s }}", k), v, -1)
+				fmt.Sprintf("${{ metadata.%s }}", k), v, -1)
```

Comment:
was this change reflected everywhere? I still see the non metadata values used in the gitHubWorkflowPermissionsTopNoWrite yaml

Reply:
+1, forgot to update. Thanks!